### DISCUSSION TITLE
Comprehensive roadmap?

### DISCUSSION THREAD

**Discussion by WillTheDeveloper** [DogOSdev/DogOS#18] [General]
Hi,

I was wondering if you are planning on building some kind of roadmap that is more detailed than the current "features" inside projects.

The way I would suggest doing the roadmap is like [this](https://github.com/Study-Portal/roadmap/projects/1) which is not originally my idea, I am doing the same way GitHub does their roadmaps. 

This just would be really helpful for new contributors to know what they could work on and contribute with.

@TaromaruYuki

**Comment by TaromaruYuki** [comment]
Hi Will!

That does seem like a good idea. I have a few [projects](https://github.com/DogOSdev/DogOS/projects) setup with stuff that needs to be worked on.

I also have a [contributing file](https://github.com/DogOSdev/DogOS/blob/main/CONTRIBUTING.md), but it's not finished. It would probably be best to move that and a lot of the readme over to GitHub wiki. 

I'll get started on this right away. Thank you for your interest.
